stacks/root.yml

```diff
@@ -103,6 +107,8 @@ Mappings:
       prefix: 15
     Play:
       prefix: 21
+    Id:
```

Comment:
Any place that you're logged in, users will have to hit ID.  So users of Publish/Castle/Adze/Metrics and eventually V3.  My thinking is that CMS and Play get hit by unauthorized requests, so they go first.  Play will likely get way more traffic than CMS, but most of that is CloudFront, so I left it 2nd here.